lab-taylor-melanie-ken/lib/bitmap-constructor.js

```diff
@@ -0,0 +1,16 @@
+'use strict';
+
+const constructor = module.exports = function(data, err) {
```

Comment:
Where did these numbers come from? I know, but would someone else looking at this know? Will you know when you look at this in a year? Store magic numbers such as these into a meaningfully named variable so that you can easily look and know what they represent.